CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## Release 11.0.0-alpha.0 (2021-05-03)
+
+[Changes since v10.1.2](https://github.com/realm/realm-studio/compare/v10.1.2...v11.0.0-alpha.0)
+
+### Breaking Changes
+
+- Added support for reading / writing Realm files using Realm file format v21 (used by realm-js v?.?.?, realm-java v?.?.?, realm-cocoa v?.?.? and realm-dotnet v?.?.?). When opening a file which is using an older format, you will get prompted to upgrade (and optionally backup) the Realm file. NOTE: Once the file has been upgraded, there is no way to downgrade it again and it can only be used by the SDKs that support Realm file format v21.
```

Comment:
```suggestion
- Added support for reading / writing Realm files using Realm file format v21 (This supports a number of new datatypes available with beta versions of realm-js v?.?.?, realm-java v?.?.?, realm-cocoa v?.?.? and realm-dotnet v?.?.?). When opening a file which is using an older format, you will get prompted to upgrade (and optionally backup) the Realm file. NOTE: Once the file has been upgraded, there is no way to downgrade it again and it can only be used by the SDKs that support Realm file format v21.
```

Reply:
Does this support any of the new datatypes (i.e. UUID) or is it just the new core?

Reply:
Great point. I've added the enhancement of being able to read / write properties of type UUID.